packages/extension/src/ui/action/utils/browser.ts

```
@@ -60,3 +60,19 @@ export const getLatestEnkryptVersion = (): Promise<string> => {
     })
     .catch(() => null);
 };
+
+export const getLatestEnkryptUpdates = (): Promise<Updates> => {
+
+  const browser = detectBrowser();
+  const url = 'https://raw.githubusercontent.com/enkryptcom/dynamic-data/main/configs/release-versions'
+  const fetchUrl = browser === BROWSER_NAMES.safari ? `${url}-safari.json` : `${url}.json`
+  console.log(fetchUrl)
+  return fetch(
+    fetchUrl
+  )
+    .then(res => res.json())
+    .then(releases => {
+      return releases;
+    })
+    .catch(() => null);
```

could log the error instead of silently dropping it
also returning null is incompatible with functions current return type